app/controllers/studentController.js

```diff
@@ -0,0 +1,28 @@
+var studentController = {
+
+   model: getmodule('app/models/studentModel'),
```

Comment:
Because you didn't add getmodule to your package.json I was not able to execute your exam:

![image](https://cloud.githubusercontent.com/assets/5324414/21127594/52036444-c0db-11e6-8699-b8201a0d3764.png)